tests/models/store.py

```diff
@@ -882,3 +886,29 @@ def test_store_create():
             name="another_store.foo",
             parent=tp.directory,
             translation_project=tp)
+
+
+@pytest.mark.django_db
+def test_store_create_templates(templates):
+    project = ProjectDBFactory(source_language=templates)
```

Comment:
yep, not sure what is best - to make it *always* defer to source_language would make this more straightforward, but it would be a change from what is there now